velox/exec/tests/utils/HiveConnectorTestBase.cpp

```diff
@@ -31,18 +31,17 @@ HiveConnectorTestBase::HiveConnectorTestBase() {
 
 void HiveConnectorTestBase::SetUp() {
   OperatorTestBase::SetUp();
-  executor_ = std::make_unique<folly::IOThreadPoolExecutor>(3);
   auto hiveConnector =
       connector::getConnectorFactory(
           connector::hive::HiveConnectorFactory::kHiveConnectorName)
-          ->newConnector(kHiveConnectorId, nullptr, executor_.get());
+          ->newConnector(kHiveConnectorId, nullptr, ioExecutor_.get());
   connector::registerConnector(hiveConnector);
   dwrf::registerDwrfReaderFactory();
 }
 
 void HiveConnectorTestBase::TearDown() {
-  if (executor_) {
-    executor_->join();
+  if (ioExecutor_) {
```

Comment:
Isn't this always non-null? Do we need to call join on driverExecutor_? Should this logic go into OperatorTestBase::TearDown?

Reply:
I don't think really need. Sorry just do code replacement. Forgot to check this.